grunttasks/serve.js

```diff
@@ -5,9 +5,15 @@
 module.exports = function (grunt) {
   'use strict';
 
-  grunt.registerTask('serve', [
-    'build:development',
-    'hapi',
-    'watch'
-  ]);
+  grunt.registerTask('serve', 'Builds the dist assets, starts the Hapi server, and watches for changes.', function (target) {
+    if (!target) {
+      target = 'development';
+    }
+
+    grunt.task.run([
+      'build:' + target,
```

Comment:
This just lets us use `npm start` which proxies `grunt serve` or call `grunt serve:production` directly if you want to build production calibre assets (minified and fingerprinted).